### PR TITLE
feat: serve styles from tileserver

### PR DESCRIPTION
**Pull request by zackpollard**

### CODE REVIEW

File: styles-gen/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "styles-gen",
```
Comment (jrasm91) [inline styles-gen/package.json:2]: should we just call this `immich-map-static` something like that instead?

**Review by jrasm91 — APPROVED** [review]

File: styles-gen/package.json (hunk 1 above)
Comment (zackpollard) [reply styles-gen/package.json:2]: This is only for the styles, so not sure. The fonts and sprites aren't handled yet but probably wouldn't be handled in this repository, as they can just be pulled statically from protomaps repo.

Comment (jrasm91) [reply styles-gen/package.json:2]: `immich-styles` seems better than `styles-gen` to me at least, but w/e you want.

Comment (zackpollard) [reply styles-gen/package.json:2]: Sure I can change it to that

**Pull request merged**